service/handler/quote_test.go

```diff
@@ -37,124 +34,64 @@ func TestUpdate(t *testing.T) {
 	}
 }
 
-// Test when fetching from an empty database
-func TestGetAllEmptyDatabase(t *testing.T) {
-	// Mock a new request
-	req, err := http.NewRequest("GET", "/v0/quote", nil)
-	if err != nil {
-		t.Errorf("failed mocking request, %v", err.Error())
-	}
-	// Initialize interactor and pass it to the handler
-	c := &entity.Config{
-		CSVFilepath: "empty.csv",
-		Port:        "8080",
-	}
-	qi := interactor.NewQuoteInteractor(*c)
-	q := NewQuote(qi)
-	// Execute request and check result
-	rec := httptest.NewRecorder()
-	q.GetAll(rec, req)
-	b, err := ioutil.ReadAll(rec.Body)
-	if err != nil {
-		t.Errorf(err.Error())
-	}
-	if strings.Compare(strings.TrimSpace(string(b)), repository.EmptyDatabase) != 0 {
-		t.FailNow()
-	}
-}
-
-// Test when fetching from a database recently populated
-func TestGetAll(t *testing.T) {
-	t.Run("TestUpdate", TestUpdate)
-	// Mock a new request
+func TestGet(t *testing.T) {
+	// We need to set some variables before starting the test tables
 	req, err := http.NewRequest("GET", "/v0/quote", nil)
 	if err != nil {
 		t.Errorf("failed mocking request, %v", err.Error())
 	}
-	// Initialize interactor and pass it to the handler
-	c := &entity.Config{
-		CSVFilepath: "test.csv",
-		Port:        "8080",
-	}
-	qi := interactor.NewQuoteInteractor(*c)
-	q := NewQuote(qi)
-	// Execute request and check result
-	rec := httptest.NewRecorder()
-	q.GetAll(rec, req)
-	if err != nil {
-		t.Errorf(err.Error())
-	}
-	if rec.Result().StatusCode != http.StatusOK {
-		t.FailNow()
-	}
-}
-
-func TestNewQuote(t *testing.T) {
+	// Initialize test tables
 	type args struct {
 		qi interactor.QuoteInteractor
 	}
 	tests := []struct {
-		name string
-		args args
-		want Quote
+		args             args
+		name             string
+		pretest          func(t *testing.T)
+		request          *http.Request
+		responseRecorder *httptest.ResponseRecorder
 	}{
-		// TODO: Add test cases.
+		// Empty database case
```

Comment:
We don't really need a comment describing the test if we can assign a name to it.